tests/conftest.py

```diff
@@ -50,3 +52,15 @@ async def database():
 async def client():
     async with AsyncClient(app=app, base_url="http://test") as client:
         yield client
+
+
+__all__ = [
+    "event_loop",
+    "recreate_database",
+    "database",
+    "client",
+    "user",
+    "category",
+    "item",
+    "access_token",
+]
```

Comment:
Why do we need this?

Reply:
I'd like to separate fixtures related to app logic into a different file. This solution doesn't seem to scale well though, do you suggest an alternative?

Reply:
Fixed